draft-ietf-capport-architecture.xml

```diff
@@ -843,8 +845,10 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
               carry any information which may mislead or misdirect the User Equipment.</li>
           <li>Even when responding to the signal, the User Equipment securely authenticates
                with API Servers.</li>
-          <li>Accesses to the API Server are rate limited, limiting the impact of a repeated
-              attack.</li>
+          <li>Accesses to the Captive Portal API are rate limited, limiting the impact of a
+              repeated attack. Because there is only one type of signal and one type of API
+              request in response to the signal, a signal-flooding attack will not obscure
+              information from the API to the User Equipment.</li>
```

Comment:
Rather than name this ("signal-flooding attack") you can just describe it.

"Any attempt to generate excessive load on either User Equipment or API by sending a large number of signals will only result in a limited number of requests for updated status."

Reply:
That's not quite the point I was trying to address. The question was about whether limiting requests could prevent information from being conveyed. I.e., by not sending a request that should be sent.
I will see if I can avoid the "signal-flooding" term, however.

Reply:
Please see the 2nd commit.